### PR TITLE
DOCS-946: Deal with camera SDK differences

### PR DESCRIPTION
**Pull request by JessamyT**
The camera API varies much more widely between languages than most of the APIs. I think we should address that in some way. I'm not sure whether this is the way--very open to other ideas but had a thought and wanted to make a suggestion!

I'll make a ticket to discuss.

### CODE REVIEW

**Comment by JessamyT** [comment]
But the API list of methods, as well as the heading here, says "GetImage," which is pretty different from "Stream"

File: docs/components/camera/_index.md [hunk 1]
```diff
@@ -145,6 +145,14 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 {{% /tab %}}
 {{% tab name="Go" %}}
 
+{{% alert title="Info" color="info" %}}
+
+Unlike most Viam [component APIs](/program/apis/#component-apis), the methods of the Go camera client do not map exactly to the Python SDK camera methods.
```
Comment (npentrel) [inline docs/components/camera/_index.md:150]:
```suggestion
Unlike most Viam [component APIs](/program/apis/#component-apis), the methods of the Go camera client do not map exactly to the names of the other SDK's camera methods.
```

File: docs/components/camera/_index.md [hunk 2]
```diff
@@ -145,6 +145,14 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
...
+`Stream` works somewhat differently from the Python `get_image` method.
+In the example below, note how a stream is constructed before the next image is gotten from the stream.
```
Comment (npentrel) [inline docs/components/camera/_index.md:152]: I'd prefer to not use Python as a reference:
```suggestion
To get an image in the Go SDK, you first need to construct a `Stream` and then you can get the next image from that stream.
```

**Review by npentrel — COMMENTED** [review]
Added two suggestions to make the note more general - let me know what you think

**Comment by JessamyT** [comment]
I like those suggestions. Do you think I should just close this PR though? I still think it could clear up confusion for some users, but I don't feel super strongly about that now. Not sure where we netted out.

**Review by npentrel — APPROVED** [review]

**Pull request merged**